src/appkit.cs

```diff
@@ -5943,6 +6150,16 @@ interface NSDrawerDelegate {
 
 	}
 
+	[Protocol]
```

Comment:
availability ?

Reply:
Same, nothing listed in header file.

Reply:
10.14 according to docs https://developer.apple.com/documentation/appkit/NSFontChanging?language=objc